helm-chart/splunk-operator/templates/deployment.yaml

```diff
@@ -108,7 +108,13 @@ spec:
 {{- toYaml . | nindent 8 }}
 {{- end }}
       terminationGracePeriodSeconds: {{ .Values.splunkOperator.terminationGracePeriodSeconds }}
-{{- with .Values.splunkOperator.volumes }}
+{{- if .Values.splunkOperator.volumes }}
       volumes:
-{{- toYaml . | nindent 8 }}
-{{- end }}
\ No newline at end of file
+{{- range $volume := .Values.splunkOperator.volumes }}
```

Comment:
Do we support this without helm?

Reply:
yes we do support. customer can use any storage class type and size.